packages/x-date-pickers/src/models/fields.ts

```diff
@@ -159,14 +156,23 @@ export type PickerFieldSlotProps<
     ref?: React.Ref<HTMLDivElement>;
   };
 
+/*
+ * Props the `useDateField()` and equivalent hooks return when used inside a single input picker.
+ * Only contains what the MUI components are passing to the field, not what users can pass using the `props.slotProps.field` and `props.slotProps.textField`.
+ */
+export type BaseSingleInputPickersFieldHooksReturnValue<
```

Comment:
Not sure about the name.
Now that we are not applying `useClearableField`, we need an interface that describes what the hooks are returning.
This will go away when we remove the fields in favor of the Base UI DX so maybe we could keep it unstable :thinking: